src/dotnet/Audio.UI.Blazor/Components/AudioPlayer/audio-player.ts

```diff
@@ -72,25 +73,35 @@ export class AudioPlayer {
             onStartPlaying: () => {
                 if (debug)
                     console.debug(`[AudioTrackPlayer #${this.id}] onStartPlaying, controllerId:${this.controller.id}`);
+                if (this.state === 'playing'){
+                    console.error("Unexpected onStartPlaying since audio player is already in playing state");
```

Comment:
let's change to `warn` - we can call this method after starving

Reply:
done